content/arduino-cloud/02.features/10.triggers/triggers.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Triggers
+description: Learn how to use triggers that allow you to send notifications based on set conditions.
+author: Hannes Siebeneicher
+tags: [IoT Cloud, Triggers, Cloud Notification]
+---
+
+Triggers react to certain conditions inside your IoT Cloud Thing, such as a boolean being true, or a string being assigned a value. As soon as a set condition is met a notification gets triggered and send to you. This is super useful when you monitor data and you need to know about any change as soon as it happens. This could be anything from different values in environmental monitoring or something security related such as movement detection.
+
+Triggers can be set up for any of your existing projects, and are found in the [cloud home section](https://cloud.arduino.cc/home/).
+
+## Hardware & Software Needed
+
+- [Arduino IoT Cloud](https://create.arduino.cc/iot/) (Maker plan).
+- Cloud-compatible boards, [see full list](https://docs.arduino.cc/arduino-cloud/getting-started/iot-cloud-getting-started#compatible-hardware).
+
+***In this tutorial, we use the [Nano 33 IoT](https://store.arduino.cc/products/arduino-nano-33-iot?queryID=undefined). This is not a requirement, you can use any IoT Cloud-compatible board for this tutorial.***
```

Comment:
I think there is an article somewhere that lists all cloud compatible boards, we could link to that here